static/js/components/ValidationAlert.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+
+import type { ValidationErrors } from '../flow/profileTypes';
+
+export default class ValidationAlert extends React.Component {
+  props: {
+    errors:   ValidationErrors,
+    children: React$Element[],
+  };
+
+  alertMessage: Function = (): React$Element|void => {
+    const { errors } = this.props;
+    if ( !_.isEmpty(errors) ) {
+      return (
+        <span className="message">
+          Please fix the errors above and try again
+        </span>
+      );
+    }
+  };
+
+  render () {
+    const { children } = this.props;
+    return (
+      <div className="validation-alert">
+        { this.alertMessage() }
+        <div className="actions">
+          { children }
+        </div>
+      </div>
```

Comment:
I don't really see why that would be more simple than what we have here. I chose to build this as a component that wraps it's children because that was the easiest way to get things to display in the layout we wanted, which is this:

```
------------
|  message |
------------
--------- ---------
| box 1 | | box 2 |
--------- ---------
```

the `actions` prop accepts either a single element or an array of elements, so if we wanted the message to appear in front of the buttons we could have done that too. I'd prefer to keep what I have here instead of building a utility class like `ValidatedDialog`.